forem/forem
# Use vcr cassette to import rss feed for two users

## What type of PR is this? (check all applicable)


## Description

A test failure was observed when the `Users::Setting.find_by(feed_url: link)` created in the before block answered nil. I suspect this was related to the feed url validation (we actually make a network request before saving the feed_url to make sure there's an xml response on the other end), and making network requests at test runtime (rather than using canned vcr responses).

Record a new cassette for this situation, which has 4 more or less identical get requests to Vaidehi's medium feed, one for each setting, and once again for the import. 

## Related Tickets & Documents

https://app.travis-ci.com/github/forem/forem/jobs/538686419

## QA Instructions, Screenshots, Recordings

I noticed that running the feeds import spec locally in main was creating unstaged files for the two `it` blocks in the last context (which now uses the included cassette).

Running the spec/services/feeds/import_spec.rb file should _not_ create new vcr cassette files in the working tree and should be possible with the network disabled.

![files that not be there](https://user-images.githubusercontent.com/1237369/134240951-6ea63524-fce9-4e21-9721-12bc0ad619d4.png)


### UI accessibility concerns?

None

## Added/updated tests?

- [x] No, and this is why: fix existing test failure

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._
